webapp/src/Service/DOMJudgeService.php

```diff
@@ -299,6 +299,21 @@ public function clearCookie(
         return $response;
     }
 
+    public function getUnreadClarifications(): array
+    {
+        $user           = $this->getUser();
+        $team           = $user->getTeam();
+        $clarifications = $team->getUnreadClarifications();
+        $unreadClarifications = [];
+        foreach ($clarifications as $clar) {
+            $unreadClarifications[] = [
```

Comment:
Should we limit them to the currently selected contest?
I.e. use
```php
$contest = $this->getCurrentContest($team->getTeamId());
```

Then check if `$contest` is null (if so, do nothing) and if not only add clarifications that match the contest cid.

Reply:
Sorry I didn't consider the case. I'll modify it.